### PR TITLE
W-12715563 Exchange add content for rpa assets jh

### PR DESCRIPTION
**Pull request by JennyHajee**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Comment by jyotirmi** [comment]
Can we make a small change in the description to include RPA assets as well? - `Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, GraphQL APIs, templates, examples, custom assets, rulesets, and integration assets such as connectors`

File: modules/ROOT/pages/index.adoc
Comment (jyotirmi) [inline modules/ROOT/pages/index.adoc:1]: From `Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, GraphQL APIs, templates, examples, custom assets, rulesets, and integration assets such as connectors`
To somthing like - Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, GraphQL APIs, templates, examples, custom assets, rulesets, `RPA assets` and integration assets such as connectors

File: modules/ROOT/pages/asset-types.adoc [hunk 1]
```diff
@@ -48,6 +48,17 @@ Configuration modules to extend the functionality of an API and enforce capabili
 * REST APIs
 +
 RAML or OAS files that specify APIs referenced by an HTTP Request connector to expose metadata to Anypoint Studio.
+
+* RPA Templates
++
+** RPA Activity Templates
++
+Activity templates are reusable code within RPA to standardize the most used activity steps and make them available to other developers in the organization. File type is .calw.
```
Comment (hannanelson) [inline modules/ROOT/pages/asset-types.adoc]:
```suggestion
Activity templates are reusable code within RPA to standardize the most used activity steps and make them available to other developers in the organization. File type is `.calw`.
```

Comment (hannanelson) [reply modules/ROOT/pages/asset-types.adoc]: (global, unless it's a huge effort)

Comment (hannanelson) [reply modules/ROOT/pages/asset-types.adoc]: SR doesn't really cover this exactly.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-filetypesandfileextensions

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ endif::[]
 :page-aliases: general::getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc about-exchange1.adoc, exchange1.adoc, exchange1-faq.adoc
```
Comment (hannanelson) [inline modules/ROOT/pages/index.adoc:6]: Why are there duplicated page-aliases lines?

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -6,7 +6,7 @@ endif::[]
...
 :page-aliases: general::getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc about-exchange1.adoc, exchange1.adoc, exchange1-faq.adoc
 
-Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, GraphQL APIs, templates, examples, custom assets, rulesets, and integration assets such as connectors. You can catalog (publish), share, discover, learn about, and reuse assets within your organization to facilitate collaboration, boost productivity, and promote standards.
+Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, GraphQL APIs, templates, examples, custom assets, rulesets, RPA assets, and integration assets such as connectors. See xref:asset-types.adoc[Asset Types] for more information about each type of asset. You can catalog (publish), share, discover, learn about, and reuse assets within your organization to facilitate collaboration, boost productivity, and promote standards.
```
Comment (hannanelson) [inline modules/ROOT/pages/index.adoc]: Two "such as". Probably ok.

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 4]
```diff
@@ -149,10 +149,40 @@ A GraphQL API schema designed with Apollo GraphQL specific directives and struct
...
-. Enter the asset portal name.
-. From the drop-down list, select the asset type *GraphQL API* .
+. Enter a name for the asset.
+. From the drop-down list, select the asset type *GraphQL API*.
 . Select which GraphQL API schema to upload, GraphQL API schema or GraphQL Apollo API schema.
-. To select the file to upload, click *Choose the file* .
+. To select the file to upload, click *Choose file*.
+. To edit the advanced settings, *GroupId*, *AssetId*, and *Version* (GAV), click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-create-an-asset.adoc:158]:
```suggestion
Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
```

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 5]
```diff
@@ -149,10 +149,40 @@ A GraphQL API schema designed with Apollo GraphQL specific directives and struct
...
+. From the drop-down list, select the asset type *RPA Activity Template*.
+. To select the .cawl file to upload, click *Choose file*.
+. To edit the advanced settings, *GroupId*, *AssetId*, and *Version* (GAV), click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
+. Select an option for *Lifecycle state*. The default is *Stable*.
+. Select *Publish*.
+
+== Create an RPA Process Template Asset
+Templates for RPA processes are imported into RPA Builder as projects and can be further customized before deploying to RPA bots. Process templates are built on common patterns for RPA to build automation faster. These templates are .crpa files that are exported from RPA Builder to share with other developers.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-create-an-asset.adoc]: Any way to avoid passive voice here?

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 6]
```diff
@@ -149,10 +149,40 @@ A GraphQL API schema designed with Apollo GraphQL specific directives and struct
 To create a GraphQL API:
 
 . In Exchange, select *Publish new asset*.
-. Enter the asset portal name.
-. From the drop-down list, select the asset type *GraphQL API* .
+. Enter a name for the asset.
+. From the drop-down list, select the asset type *GraphQL API*.
 . Select which GraphQL API schema to upload, GraphQL API schema or GraphQL Apollo API schema.
-. To select the file to upload, click *Choose the file* .
+. To select the file to upload, click *Choose file*.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-create-an-asset.adoc]: For these two that start with "To select", I sort of want to flip to have the imperative start the step.
Click ... to ...

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 7]
```diff
@@ -162,10 +192,12 @@ Exchange generates the group ID, asset ID, and version for you, and you can chan
...
 
+To create a ruleset asset:
+
 . In Exchange, select *Publish new asset*.
-. Enter the asset portal name.
-. From the drop-down list, select the asset type *Ruleset* .
-. To select the file to upload, click *Choose the file* .
+. Enter a name for the asset.
+. From the drop-down list, select the asset type *Ruleset*.
+. To select the file to upload, click *Choose file*.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-create-an-asset.adoc]: (see previous similar comment)

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/to-create-an-asset.adoc (hunk 6 above)
Comment (JennyHajee) [reply modules/ROOT/pages/to-create-an-asset.adoc]: Hi, those steps are optional. I see now that they should start with a conditional "If". I will fix this.

Comment (JennyHajee) [reply modules/ROOT/pages/to-create-an-asset.adoc]: I will fix this globally for all instances of optional steps.

File: modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (JennyHajee) [reply modules/ROOT/pages/index.adoc]: I will rewrite this intro.

File: modules/ROOT/pages/to-create-an-asset.adoc (hunk 5 above)
Comment (JennyHajee) [reply modules/ROOT/pages/to-create-an-asset.adoc]: Does this work?
Templates for RPA processes are imported into RPA Builder as projects and are customizable before deploying to RPA bots.

Comment (JennyHajee) [reply modules/ROOT/pages/to-create-an-asset.adoc]: Rewrote the sentence to be active...I think.

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (JennyHajee) [reply modules/ROOT/pages/index.adoc:6]: removed duplication

File: modules/ROOT/pages/asset-types.adoc (hunk 1 above)
Comment (JennyHajee) [reply modules/ROOT/pages/asset-types.adoc]: fixed

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (JennyHajee) [reply modules/ROOT/pages/index.adoc:6]: fixed

**Review by hannanelson — APPROVED** [review]

**Pull request merged**